### PR TITLE
Update French/English strings

### PR DESCRIPTION
**Pull request by clarfonthey**
Adds strings for the new stuff in pre-release 0.1.0 for French, and slightly modifies the English strings to be more natural; the setting is in sentence case instead of title case, and the "Set the" has been dropped.

### CODE REVIEW

**Comment by mal0ki** [comment]
Since I can't really read enough French to judge this, I tagged in some others for reviews

File: config/locales/fr.yml [hunk 1]
```diff
@@ -404,6 +404,12 @@ fr:
       mascot:
         desc_html: Affiché sur plusieurs pages. Au moins 293×205px recommandé. Lorsqu’il n’est pas défini, retombe à la mascotte par défaut
         title: Image de la mascotte
+      max_bio_chars:
+        desc_html: 'Définir la taille maximum de biographies en charactères (défaut : 500)'
```
Comment (Feufochmar) [inline config/locales/fr.yml]: 'Longueur maximale des biographies en nombre de caractères (500 par défaut)'
I would drop the verb 'Définir' if it's not on a button or a link.

File: config/locales/fr.yml [hunk 2]
```diff
@@ -404,6 +404,12 @@ fr:
...
+        title: Taille Maximum de Biographies
```
Comment (Feufochmar) [inline config/locales/fr.yml]: 'Longueur maximale des biographies'

File: config/locales/fr.yml [hunk 3]
```diff
@@ -404,6 +404,12 @@ fr:
...
+      max_toot_chars:
+        desc_html: 'Définir la taille maximum de pouets en charactères (défaut : 500)'
```
Comment (Feufochmar) [inline config/locales/fr.yml]: 'Longueur maximale des pouets en nombre de caractères (500 par défaut)'

File: config/locales/fr.yml [hunk 4]
```diff
@@ -404,6 +404,12 @@ fr:
...
+        title: Taille Maximum de Pouets
```
Comment (Feufochmar) [inline config/locales/fr.yml]: 'Longueur maximale des pouets'

**Comment by Feufochmar** [comment]
I commented the commit with better wordings, though I need to check the context where the strings appear.

File: config/locales/fr.yml (hunk 1 above)
Comment (clarfonthey) [reply config/locales/fr.yml]: In English it was "Set the maximum biography size (default: 500)" so I went with a more direct translation. I agree with you, though; I'll update both English and French.

**Comment by clarfonthey** [comment]
The maximum sizes are for the admin settings, and the other string is for user settings.

**Comment by Feufochmar** [comment]
I've checked the admin settings page, what I suggested seems right in that context.

**Comment by clarfonthey** [comment]
Updated! Also added changes in the PR description for others reading.

**Review by mal0ki — APPROVED** [review]
Approved based on other's feedback.

**Review by 1011X — APPROVED** [review]

**Pull request merged**